### DISCUSSION TITLE
Steps to enable ssl config using letsencrypt

### DISCUSSION THREAD

**Discussion by swapnil28490** [requarks/wiki#3486] [Help / Questions]
**Question**
How to enable ssl config using letsencrypt ?
Please provide the step to configure letsencrypt.

I changed the paramters as below : 
*****************************************
ssl:
enabled: false
  port: 3443

  # Provider to use, possible values: custom, letsencrypt
  provider: letsencrypt

# ++++++ For letsencrypt only ++++++
  domain: wiki.****.com
  subscriberEmail: *******@gmail.com
************************************************************
It didnt work getting error as below: 
***************************** ERROR ***************************************
warn: (LETSENCRYPT) Error: Error: Failed HTTP-01 Pre-Flight / Dry Run.
Expected: 'test-521dca998d13981a7abe932ce917088b-0.-CqucwPuVR3wq_hwubH4AKpejWCaqxAOE7DlMUDKKNo'
Got: '<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /.well-known/acme-challenge/test-521dca998d13981a7abe932ce917088b-0 was not found on this server.</p>
</body></html>
'
See https://git.rootprojects.org/root/acme.js/issues/4
2020-11-16T12:29:11.711Z [MASTER] error: Error: Failed HTTP-01 Pre-Flight / Dry Run.
************************ ERROR *****************************************************

**Host Info (please complete the following information):**
OS: [CentOS 7]
Wiki.js version: [2.5.170]
Database engine: [e.g. postgres 11]

Thanks in advance

**Comment by pwnyprod** [comment]
i think you missed some config args, lookup in the docu: https://docs.requarks.io/en/install/config#lets-encrypt

**Comment by swapnil28490** [comment]
I have followed all the steps mentioned in the document.

**Comment by Smankusors** [comment]
is your wiki accessible on the internet? It seems the one that replied not found is not from Wiki.js. Is your Wiki.js run on HTTP port 80 or not?
